.github/workflows/pull-request.yml

```diff
@@ -123,6 +124,12 @@ jobs:
             ISSUE_GUIDELINES.md
             PR_GUIDELINES.md
             README.md
+            *.pem
+            *.key
```

Comment:
Why were these added? Why would we be storing cryptography and cybersecurity related files in the repository?

Reply:
it was present in pull-request.yml file, this changes was reverted

![Uploading image.png…]()

Reply:
OK. They are not necessary, but let them remain.

Reply:
alright sir